src/test/custom_nodes/node_add_sub.c

```diff
@@ -77,17 +77,33 @@ int execute(const struct CustomNodeTensor* inputs, int inputsLength, struct Cust
     return 0;
 }
 
-int releaseBuffer(struct CustomNodeTensor* output) {
-    printf("CUSTOM ADD_SUB NODE => Releasing Buffer (output name: %s)\n", output->name);
-    fflush(stdout);
-    free(output->data);
-    free(output->dims);
+int getInputsInfo(struct CustomNodeTensorInfo** info, int* infoLength, const struct CustomNodeParam* params, int paramsLength) { 
+    *infoLength = 1;
+    *info = (struct CustomNodeTensorInfo*) malloc (*infoLength * sizeof(struct CustomNodeTensorInfo));
+    (*info)->name = "input_numbers";
+    (*info)->dimsLength = 2;
+    (*info)->dims = (uint64_t*) malloc((*info)->dimsLength * sizeof(uint64_t));
+    (*info)->dims[0] = 1;
+    (*info)->dims[1] = 50;
```

Comment:
In already written test you use this node with input size (1,3) I think. When you will finish config validation using metadata from this MR you will have to rewrite those tests to use (1,50) r change metadata here I think

Reply:
You are right, execution tests can be more flexible because these do not (yet) check custom node metadata at runtime.

Reply:
Add todo in code

Reply:
Done